src/redux/slices/homeSlice.ts

```diff
@@ -0,0 +1,95 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+/*
+  Home page has multiple UI components.
+  This slice will determine whether
+  Menu is open?
+  Search is shown?  //Maybe I should use navigate() to just change to the search page. Search is kind of like a separate page, not on home page?
+  Basket Tab at bottom of screen is shown?  //this state is dependent on cartState. if(cartState.items[].length !== 0) basketItemExists = true; cartSlice not yet implemented.
+  Order history icon is shown?  //After customer places order, icon on homepage will appear to view order history.
+
+  no PayloadAction for any of the reducers. Just toggling true/false.
+*/
+
+export interface HomeState {
+  menuShow: boolean;
+  searchShow: boolean;
+  basketItemExists: boolean;
+  orderHistoryExists: boolean;
+}
+
+const initialState: HomeState = {
+  menuShow: false,
+  searchShow: false,
+  basketItemExists: false,
+  orderHistoryExists: false,
+};
+
+const authSlice = createSlice({
```

Comment:
change to homeSlice

Reply:
As discussed on telegram, the homeUIState will not be on redux persisted store. Instead, I have created a HomeProvider and HomeContext using useContext and useReducer. This will manage state for the homepage.
The files are in the Redux folder, which is kind of misleading. The folder structure there could use some work.

Reply:
I meant the variable name. The file is homeSlice so the variable should also be called homeSlice